pom.xml

```diff
@@ -16,23 +16,23 @@
   <properties>
     <maven.compiler.source>21</maven.compiler.source>
     <maven.compiler.target>21</maven.compiler.target>
-    <spring.boot.version>3.3.3</spring.boot.version>
+    <spring.boot.version>3.3.5</spring.boot.version>
     <jib-maven-plugin.version>3.4.2</jib-maven-plugin.version>
-    <test-containers.version>1.20.1</test-containers.version>
-    <maven-surefire.version>3.4.0</maven-surefire.version>
+    <test-containers.version>1.20.3</test-containers.version>
+    <maven-surefire.version>3.5.1</maven-surefire.version>
     <maven-surefire-plugin.version>${maven-surefire.version}</maven-surefire-plugin.version>
     <maven-failsafe-plugin.version>${maven-surefire.version}</maven-failsafe-plugin.version>
     <commons-text.version>1.12.0</commons-text.version>
-    <wiremock.version>3.9.1</wiremock.version>
+    <wiremock.version>3.9.2</wiremock.version>
 
     <skip.unit.tests>false</skip.unit.tests>
     <skip.integration.tests>false</skip.integration.tests>
 
     <!--companies house-->
-    <structured-logging.version>3.0.14</structured-logging.version>
-    <sdk-manager-java.version>3.0.3</sdk-manager-java.version>
-    <private-api-sdk-java.version>4.0.217</private-api-sdk-java.version>
-    <kafka-models.version>3.0.4</kafka-models.version>
+    <structured-logging.version>3.0.20</structured-logging.version>
+    <sdk-manager-java.version>3.0.9</sdk-manager-java.version>
+    <private-api-sdk-java.version>4.0.234</private-api-sdk-java.version>
+    <kafka-models.version>3.0.8</kafka-models.version>
 
     <!--explicit versions of transitive dependencies with vulnerabilities in previous versions-->
     <logback.version>1.4.14</logback.version>
```

Comment:
logback.version and jackson-databind.version aren't used anywhere, can be removed.

Reply:
Addressed in latest commit